ts/main.ts

```diff
@@ -43,5 +43,109 @@ $entryForm.addEventListener('submit', (event: Event): void => {
   data.entries.unshift($formData);
   if (!$image) throw new Error('the img query failed');
   $image.src = 'images/placeholder-image-square.jpg';
-  $entryForm.reset();
+  $form.reset();
+  $ulElement.prepend(renderEntry(data.entries[0]));
```

Comment:
Rather than doing `data.entries[0]` I would do `$formData`. While in theory it should be the first object at the beginning of the array, this way we guarantee that we're passing in the object we created